src/napari/_qt/_qapp_model/qactions/_layerlist_context.py

```diff
@@ -68,6 +68,13 @@ def _copy_spatial_to_clipboard(layer: Layer) -> None:
     )
 
 
+DEFAULT_1D_VALUES = {
+    'scale': 1.0,
+    'translate': 0.0,
+    'units': 'px',
+}
```

Comment:
Looking at the tests, I'm confused why there's no shear/rotate defaults?

Reply:
Because it depends on data dimensionality. We may provide a callable that will take `ndim` as parameter.

Reply:
No. This one are 1d vectors. 
Others are 2d matrices where not all fields are filled with same value. 
Shear is the upper triangle, where the default rotation is the identity matrix.